src/components/announcer/CyVerseAnnouncer.js

```diff
@@ -34,6 +41,18 @@ class CyVerseAnnouncer extends Component {
     dequeue = () => {
         if (msgQueue.length > 0) {
             this.setState({ msg: msgQueue.shift(), open: true });
+            if (
+                msgQueue.length === 0 &&
+                this.state.timeout !== EMPTY_QUEUE_TIMEOUT
```

Comment:
Seems like there's a typo here.  I don't think this conditional will ever be true since it's inside the above condition which checks if the queue length is greater than 0 🤔

Reply:
I think the `msgQueue.shift()` in line 43 can potentially remove the last element of the list and then the length will be 0

(sorry, saw the comment pop up in the github slack channel above your review of my PR, got curious, and had to look :laughing: )

Reply:
Ah, you're right, I missed that 😄

Reply:
(and this is why functional programming doesn't like mutable data structures!

woo~)